grc/gui/Application.py

```diff
@@ -490,6 +492,14 @@ def flow_graph_update(fg=flow_graph):
             action.save_to_preferences()
             for page in main.get_pages():
                 flow_graph_update(page.flow_graph)
+        elif action == Actions.TOGGLE_SHOW_VARIABLE_PARAMS_NAME:
+            action.set_active(not action.get_active())
+            action.save_to_preferences()
+            flow_graph_update()
+        elif action == Actions.TOGGLE_SHOW_VARIABLE_PARAMS_VALUE:
+            action.set_active(not action.get_active())
+            action.save_to_preferences()
+            flow_graph_update()
```

Comment:
What happens if both are `False`? Maybe ticking one off, enables the other? That way 3 states are possible.

Reply:
right now it reverts to the default behavior, which I think is the right thing to do.  I don't have a good concept for how it would work with the 'full coverage' of options (like a checkbox for "show both", "show vals", and "show params" that all automatically check/uncheck eachother?)